lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/api/VortexFuture.java

```diff
@@ -142,4 +205,13 @@ public void run() {
     }
     this.countDownLatch.countDown();
   }
+
+  /**
+   * Called by VortexMaster to let the user know that the task was cancelled.
+   */
+  @Private
+  public void cancelled() {
```

Comment:
Can multiple threads call this method at the same time?
If so, do we have to synchronize it? Or do we have to make `cancelled` `AtomicBoolean` at least?